client/cruise.lua

```diff
@@ -76,14 +76,13 @@ local function TriggerCruiseControl()
 end
 
```

Comment:
```suggestion

lib.onCache('ped', function(value)
    Player = ped
end)

```
You can update the Player this way as well instead of only in the command, if the ped changes after executing the command, it won't work until the command is executed again, also rename `Player` to `playerPed` to not overwrite the Player statebag method

Reply:
Renamed to PlayerPed, it was Player by default before

Reply:
Add the ped update as well